create_certificates.sh

```diff
@@ -0,0 +1,15 @@
+echo "Generating web interface certs..."
+cd certificates
```

Comment:
Now that we've moved this script to this repo, its possible that users will run it from a different folder (e.g., the workspace root). I'd recommend adding a [check for whether the `certificates` director exists](https://sentry.io/answers/determine-whether-a-directory-exists-in-bash/), as a heuristic for whether the user is in the `stretch_web_teleop` folder. If it does not exist, the script should echo a message telling users it must be run from `stretch_web_teleop`, and exit.